### PR TITLE
Increase coverage of acceptance tests github workflow

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

adds more testing to PR acceptance tests

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed
- [X] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite

- Cucumber tests were added

- [X] **DONE**

## Links
fixes:

- Add build host to the infra: SUSE/spacewalk#24433
- Do not schedule acceptance tests runs on forks: SUSE/spacewalk#23654
- Add debian minion: SUSE/spacewalk#23593
- Fix service notifications tests: SUSE/spacewalk#20962
- Fix openscap tests: SUSE/spacewalk#20958
- Fix ansible tests: SUSE/spacewalk#20955
- Fix monitoring secondary feature: SUSE/spacewalk#20670
- Fix scc: SUSE/spacewalk#20954
- Fix OS image build secondary feature: SUSE/spacewalk#20678
- Fix docker api secondary feature: SUSE/spacewalk#20677
- Fix monitoring secondary feature: SUSE/spacewalk#20670

Ports:
- Manager-5.0 https://github.com/SUSE/spacewalk/pull/25554

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
It needs https://github.com/uyuni-project/uyuni/pull/9288

**Comment by jordimassaguerpla** [comment]
All this code was already reviewed here


CI: skip scheduled actions on forks #8551
CI: enable deblike_minion tests #8554
CI: enable srv notification tests for github actions #8552
CI: enable openscap test #8553
CI: fix ansible and prometheus tests #8562
ci: add ansible package in sle minion #8566
enable os build image #8568
CI: Add container registry #8811
CI: Add and enable non-authorized container registry #8813

**Comment by jordimassaguerpla** [comment]
This BIG PR is the sum of smaller PRs which were already reviewed

**Comment by jordimassaguerpla** [comment]
Tests passed once.

**Comment by jordimassaguerpla** [comment]
@srbarrios can you review it again? Ignore the "TEST" commit. I will remove that one.

**Comment by jordimassaguerpla** [comment]
tests passed twice

**Comment by jordimassaguerpla** [comment]
tests passed three consecutive times. This is stable.

**Comment by jordimassaguerpla** [comment]
> @srbarrios can you review it again? Ignore the "TEST" commit. I will remove that one.

I removed the TEST commit.

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016-2023 SUSE LLC
```
Comment (srbarrios) [inline testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:1]: 2024

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Licensed under the terms of the MIT license.
+
+@buildhost
+Feature: Bootstrap a build host via the GUI
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
```
Comment (srbarrios) [inline testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:8]: It does not seems we need admin rights here

**Review by srbarrios — APPROVED** [review]

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature (hunk 2 above)
Comment (jordimassaguerpla) [reply testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:8]: We have to login with some user, or? Which other user could I use?

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature (hunk 1 above)
Comment (jordimassaguerpla) [reply testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:1]: done.

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2016-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@buildhost
+Feature: Bootstrap a build host via the GUI
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
```
Comment (srbarrios) [inline testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:8]:
```suggestion
    Given I am authorized
```

File: testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature (hunk 2 above)
Comment (srbarrios) [reply testsuite/features/github_validation/init_clients/buildhost_bootstrap.feature:8]: Suggested below.

**Review by srbarrios — DISMISSED** [review]

**Comment by jordimassaguerpla** [comment]
adding test commit to pass the tests after the last change

**Comment by jordimassaguerpla** [comment]
Tests pass. Removing test commit.

**Pull request merged**